modules/caddyhttp/reverseproxy/healthchecks.go

```diff
@@ -90,6 +90,12 @@ type ActiveHealthChecks struct {
 	// this value is ignored.
 	Port int `json:"port,omitempty"`
 
+
+	// Configures the method of transport for the active health checker. 
+	// The default transport is the handler's transport
+	TransportRaw json.RawMessage `json:"transport,omitempty"`
+	Transport http.RoundTripper `json:"transport,omitempty"`
```

Comment:
I'd love for it to be unexported, but I haven't inspected the code well enough to see it doable.

```suggestion
	Transport http.RoundTripper `json:"-"`
```

Reply:
fair, transport is now unexported.

Reply:
okay we can make it be Unexported. updated